src/pages/public/login/index.js

```diff
@@ -9,8 +9,9 @@ const Login = () => {
   const localData = JSON.parse(localStorage.getItem("user"));
 
   const logInSubmit = (event) => {
-    (localData.some((item) => { return ((MailId == item.Email) && (Password == item.Password)) })) ? (alert("logIn Successful"), AppDispatcher.updateUserTokens()) : (alert('Email or Password is wrong'), event.preventDefault());
```

Comment:
Try to use Prettier (or use Document Format) or add css to make short lines of code. Very long codes are not easily readable.